src/main/steps/applicant1/hub-page/sole/hub.njk

```diff
@@ -1,11 +1,30 @@
 {% from "govuk/components/details/macro.njk" import govukDetails %}
 
-{% if progressionIndex == 9 %}
+{% set readMoreContent %}
+  <p class="govuk-body">{{ readMoreSummary }}</p>
+  <ol class="govuk-list govuk-list--number">
+    {% for property, line in readMoreSteps %}
+      <li>{{ line | safe }}</li>
+    {% endfor %}
+  </ol>
+  <p class="govuk-body">{{ holding.line4 | safe }}</p>
```

Comment:
if that `holding.line4` is used multiple times in the template outside of the Holding state content it might be worth just making that a separate variable instead of leaving it inside the Holding content